runtime/moonbase/src/lib.rs

```diff
@@ -1169,7 +1169,7 @@ construct_runtime! {
 			pallet_collective::<Instance3>::{Pallet, Call, Storage, Event<T>, Origin<T>, Config<T>} = 40,
 		ConvictionVoting: pallet_conviction_voting::{Pallet, Call, Storage, Event<T>} = 41,
 		Referenda: pallet_referenda::{Pallet, Call, Storage, Event<T>} = 42,
-		Origins: pallet_custom_origins::{Origin} = 43,
+		Origins: pallet_governance_origins::{Origin} = 43,
```

Comment:
Would there be any advantage to abandoning `43` and giving this an unused index?

Reply:
What is the advantage? It is the same pallet, it was just moved into our `pallets/` folder and renamed.

Reply:
I'm not saying there is, just thought it was worth a quick discussion.